index.html

```diff
@@ -22,6 +22,32 @@
       font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
       line-height: 1.5;
     }
+    a {
+        color:#222;
+    }
+    h2 {
+        display:inline;
+        padding-right:10px;
+    }
+    .social-nav{
+        padding-left:0;
+    }
+    .social-nav li {
+        display:inline;
+        padding-right:10px;
+    }
+    .social-nav li img {
+        width:30px;
+    }
+    .social-nav li img:hover {
+        transform: scale(1.5);
+    }
+    img.new-tab {
+        width:20px;
+    }
+    @media (prefers-color-scheme: dark) {
+        a:hover {font-size:18px;}
+    }
```

Comment:
What's the reasoning behind enlarging links on hover in dark mode?

Reply:
I thought adding a hover effect for those in dark mode would make the links stand out more

Reply:
Removed this for now. It's causing a layout shift on link hover. Feel free to open another PR to tackle this for both light and dark modes.